js/src/routes.js

```diff
@@ -60,6 +62,7 @@ const contractsRoutes = [
 ];
 
 const settingsRoutes = [
+  { path: 'advanced', component: SettingsAdvanced },
```

Comment:
The issue is that we now have 2 tabs with basically nothing, hence adding the extra stuff (e.g. language selector as an example as well features PR extends the "Parity" tab.) 

So let's stick with what we have instead of throwing more tabs at it. (We know it is difficult to combine when things get out of hand, eg. Accounts buttons) We can always split the UI-specific stuff at a later point when it becomes unwieldly.

The point below stands - while this is really just for dev and not in wide use, disable the selection in dev mode. (When we bring in Features as per open PR, we can set when/where we want it visible. Quite happy to have it showing, but not while it covers one small modal and not more.)